features/language-support.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Language Support
+  In order to know whether or not Compensated will work in my ecosystem
+  I would like to know which language versions each Compensated package supports
+
+  @no-ci
```

Comment:
Should we remove this?

Reply:
I don't think so? [CI already runs against Ruby 2.4~2.7](https://github.com/zinc-collective/compensated/blob/0.X/.travis.yml#L5-L8). Maybe we could add a build action that _only_ runs this, and runs it against a number of operating systems?

Reply:
I was under the impression that we'd want to enable this test so both CI and developer local machine have a single source of truth on what are the supported Ruby version.

I think it'd be nice to eventually to just update one place to indicate which Ruby version we are supporting.

Reply:
I agree, however if we were to do that then our CI times would go from ~1m to ~15m; because it would add compiling and installing Ruby. This gives us a single source-of-truth for what we _intend_ to support; and maybe there's a way to automate making this match the .travis.yml file; but that seems like a lot of effort for little pay-off.